### PR TITLE
responseListener.listenForResponseFromTransaction

### PR DESCRIPTION
**Pull request by KuphJr**
Use new `responseListener.listenForResponseFromTransaction` method from the latest Functions Toolkit release

### CODE REVIEW

File: tasks/Functions-consumer/request.js [hunk 1]
```diff
@@ -206,15 +206,14 @@ task("functions-request", "Initiates an on-demand request from a Functions consu
     }
 
     // Listen for fulfillment
-    const requestId = requestTxReceipt.events[2].args.id
     spinner.start(
-      `Request ${requestId} has been initiated. Waiting for fulfillment from the Decentralized Oracle Network...\n`
+      `Request ${requestTxReceipt.events[2].args.id} has been initiated. Waiting for fulfillment from the Decentralized Oracle Network...\n`
```
Comment (bolekk) [inline tasks/Functions-consumer/request.js:210]: I changed other examples to print a transaction hash here and the request ID only after fulfillment. Maybe do the same here?

File: networks.js [hunk 2]
```diff
@@ -101,7 +101,7 @@ const networks = {
     accounts,
     verifyApiKey: process.env.POLYGONSCAN_API_KEY || "UNSET",
     chainId: 80001,
-    confirmations: DEFAULT_VERIFICATION_BLOCK_CONFIRMATIONS,
+    confirmations: DEFAULT_VERIFICATION_BLOCK_CONFIRMATIONS * 4,
```
Comment (bolekk) [inline networks.js:104]: That looks a little weird. Does it actually hurt to keep it at DEFAULT?

Comment (zeuslawyer) [reply networks.js:104]: do we want to have network specific defaults?

File: tasks/Functions-consumer/request.js (hunk 1 above)
Comment (zeuslawyer) [reply tasks/Functions-consumer/request.js:210]: Is there informational value in having both? something along the lines of:

`RequestId ${reqId} is waiting for fulfillment from the Decentralized Oracle Network.  Track this fulfillment by its Tx hash ${txHash}. `

**Review by zeuslawyer — COMMENTED** [review]
LGTM

**Review by pinebit — APPROVED** [review]

File: networks.js (hunk 2 above)
Comment (KuphJr) [reply networks.js:104]: I'll just set this to 10 blocks. I've noticed that it can be a problem if we don't wait long enough before verifying contracts.

Comment (bolekk) [reply networks.js]: 10 blocks on Sepolia is 2 minutes, way too slow IMO

**Pull request merged**